src/js/components/Image/__tests__/__snapshots__/Image-test.js.snap

```diff
@@ -1,5 +1,32 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Image fill renders 1`] = `
+.c0 {
+  font-size: 18px;
+  line-height: 24px;
+  box-sizing: border-box;
+  -webkit-text-size-adjust: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+}
+
+.c1 {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+<div
+  className="c0"
+>
+  <img
+    className="c1"
+    fill={true}
```

Comment:
We shouldn't be leaking the `fill` into the DOM. We handle this in other areas by renaming the prop in Image to something like `fillProp` and passing that to StyledImage.

Reply:
@Mr-Magnificent we shouldn't have our props to appear on the DOM in order to keep the DOM nice and clean, can you please take a look at this last comment?